core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -95,11 +96,15 @@ class LoadBalancerService(config: WhiskConfig, verbosity: LogLevel)(
                 info(this, s"Completion counts: [${invokerCounts.mkString(", ")}]")
                 info(this, s"Invoker health: [${health.mkString(", ")}]")
             }
-            Map(LoadBalancerKeys.invokerHealth -> getInvokerHealth()) ++
-                getUserActivationCounts()
+            Map(LoadBalancerKeys.invokerHealth -> getInvokerHealth())
         })
 
     /**
+     * A bridge method to impedance match against RestAPI.
+     */
+    def acceptRequest(lbr: LoadBalancerReq) = publish(lbr._1)(lbr._2)
```

Comment:
Is this really needed? I think we should get rid of `LoadBalancerReq` and this `acceptRequest` method.

Depending on which level we want to have the abstraction:
1. (less intrusive): remove the types + method as mentioned and replace them with explicit calls. The line  167 in RestAPI.scala becomes `protected implicit val performLoadBalancerRequest = loadBalancer.publish(_: ActivationMessage)(_: TransactionId)` for example.
2. (more intrusive, but not really): just remove the abstraction layer and pass the LoadBalancerService through, as we do for the EntitlementService anyway.

Thoughts?